buildSrc/src/main/kotlin/PreplaceTask.kt

```
@@ -0,0 +1,71 @@
+  fun doTaskAction() {
+    // check if perl is available
+    var isPerlInstalled = true
+    try {
+      val myStandardOutput = ByteArrayOutputStream()
+      val myErrorOutput = ByteArrayOutputStream()
+      execOperations.exec {
+        commandLine("perl", "-version")
+        standardOutput = myStandardOutput
+        errorOutput = myErrorOutput
+      }
+    } catch (ex: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)